rosdep/base.yaml

```diff
@@ -727,6 +727,15 @@ espeak:
   fedora: [espeak]
   gentoo: [app-accessibility/espeak]
   ubuntu: [espeak]
+exfat-fuse:
+  debian: [exfat-fuse]
+  gentoo: [fuse-exfat]
```

Comment:
@allenh1 should the gentoo be `sys-fs/` prefixed here and below?

Reply:
@tfoote yes, that would be my preference.

Reply:
Got it, addressed.